staff/maryam-malek/cv/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+   
+</head>
+<body>
+        <header>
+            <h1 id="name">Richard House</h1>
+            <address>
+                Email adress: <br> 
+                <a href="mailto:webmaster@example.com">webmaster@example.com</a><br> 
+                Portafolio:<br>
+                <a href="mailto:webmaster@example.com">example.com</a>.<br>
+                Street beta<br>
+                London
```

Comment:
you should have put the ```a``` inside a ```p``` alongside its definition like so:
```
<p>Email adress: <a href="mailto:webmaster@example.com">webmaster@example.com</a></p>
```
also, the ```br``` are not needed

Reply:
So it's better to put all the lines inside p tags? Like this
```
 <p>Email adress:<br><a href="mailto:webmaster@example.com">webmaster@example.com</a></p>
 <p>Portafolio<br><a href="mailto:webmaster@example.com">example.com</a></p>
 <p>Street beta</p>
 <p>London</p>
```

Reply:
definetely :)